.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/KataGo-pytorch-rewrite"]
+	path = submodules/KataGo-pytorch-rewrite
```

Comment:
Ah originally I named it this because I was using this branch https://github.com/lightvector/KataGo/tree/pytorch-rewrite that hadn't been merged into the master KataGo branch. But it was just merged this week so I renamed the submodule to just KataGo.